code/modules/admin/holder2.dm

```diff
@@ -145,6 +145,29 @@ you will have to do something like if(client.holder.rights & R_ADMIN) yourself.
 		return 1
 	return 0
 
+/**
+ * Requires the holder to have all the rights specified
+ *
+ * rights_required = R_ADMIN|R_EVENT means they must have both flags, or it will return false
+ */
+/proc/check_rights_multiple(rights_required, show_msg = TRUE, mob/user = usr)
```

Comment:
I'd suggest changing this to be called something like `check_rights_all`, so it's clear that you need every bit of access.